src/plugins/piral-oidc/src/types.ts

```diff
@@ -215,3 +225,22 @@ export enum OidcErrorType {
 export interface PiralOidcError extends Error {
   type: Readonly<OidcErrorType>;
 }
+
+export interface SignInRedirectParams {
+  /**
+   * Values used to maintain state between the sign in request and the callback.
+   * These will be available on the result from the handleAuthentication function
+   * successfully authenticates from a callback state.
+   */
+  state?: PiralCustomRedirectState;
+}
+
+/** Result that is returned from the handleAuthentication function */
+export interface AuthenticationResult {
+  /** Whether or not the application should be rendered */
+  shouldRender: boolean;
+  /** The request state that is returned from any callbacks.
+   * This will only be populated if a callback method is called.
+   */
+  state?: PiralCustomRedirectState;
```

Comment:
Same question as above.

Reply:
Resolved, `state` is now typed as  `any`